### PR TITLE
Improve project setup long-description

### PR DESCRIPTION
**Pull request by ikkoham**
### Summary

Change the long description to README.md as well as Terra.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -15,9 +15,6 @@
 import sys
 import os
```
Comment (mrossinek) [inline setup.py:16]:
```suggestion
import os
import re
```

**Review by mrossinek — DISMISSED** [review]
I am in favor of this change :+1:

**Comment by ikkoham** [comment]
@woodsp-ibm Thanks. I fixed the link and I also fixed the broken links. (Qiskit/qiskit has master branch not main.)

**Review by mrossinek — APPROVED** [review]
Thanks @ikkoham!

**Review by woodsp-ibm — APPROVED** [review]
Thanks for taking care of the links

**Pull request merged**